operations/helm/charts/grafana-agent/values.yaml

```diff
@@ -5,6 +5,14 @@ nameOverride: null
 # resource names.
 fullnameOverride: null
 
+global:
```

Comment:
Can this get a comment explaining what it is and how it works?

I understand that it's convention with a few Helm charts, but I think the way these Helm charts use global sections is the inverse from what I would normally expect; I would normally expect global sections to provide the defaults, and then for other settings to override the global defaults.

Since this works in the opposite direction (globals override other settings), I'd like to add a comment making that clear.

Reply:
I added a comment above global. For information, it works that way because in most of the non grafana upstream projects I've seen, the image registry is generally something other than docker like quay so the existing values are  already defined as:

```yaml
image:
  registry: quay.io
```
That means that if the "local" setting was taking precedent then the global value would never be applied.